book/animations.md

```diff
@@ -1743,8 +1747,8 @@ Here is how the opacity transition example's composited layers should look
 [flag]: https://docs.python.org/3/library/argparse.html
 
 [^flag-recommendation]: I also recommend you add a mode to your browser that
-disables compositing (i.e. setting `needs_compositing` to `False` for every
-`VisualEffect`), and disables use of the GPU (i.e. going back to the old way of
+disables compositing (i.e., setting `needs_compositing` to `False` for every
+`VisualEffect`), and disables use of the GPU (i.e., going back to the old way of
```

Comment:
These two (plus one in the glossary) are the only use of `i.e.` in the book. Can we replace with "that is"?

Reply:
I also looked for `e.g.` and there a few more of those that look harder to replace. But "i.e." here is very replacable.